### PR TITLE
Support for Google Analytics

### LINKED ISSUE #98: Google analytics
Just wondering if we could get proper GA integration with Angular? Might be as simple as leaning on [Angulartics.](http://angulartics.github.io/)

### PR DESCRIPTION
**Pull request by longhotsummer**
We have to explicitly tell Angulartics to send a pageview when the search term changes in /search. I don't know if we should be doing the same thing in other views (eg. collections search?) or why we need to do that explicitly.

Closes #98.

### CODE REVIEW

**Comment by pudo** [comment]
I'm not allowed to use GA in my env; is there any way you could also put this behind a config part, at least the template integration?

**Comment by longhotsummer** [comment]
If you don't declare GA_TRACKING_ID in the config, the GA script is not injected at all (and the angular code just does nothing). That's why the example config file has it commented out. Is that sufficient?

**Comment by longhotsummer** [comment]
Does this cover your use case, @pudo ?

**Comment by pudo** [comment]
Sorry I still wanted to check out if there's a way this can be made to support Piwik also. I'm not allowed to use GA (or load their assets -- which this currently does?), but more fine-grained analytics would be great.

File: aleph/templates/layout.html [hunk 1]
```diff
@@ -125,6 +125,19 @@
       </div>
     </footer>
 
+    {% if config.GA_TRACKING_ID is defined %}
```
Comment (longhotsummer) [inline aleph/templates/layout.html]: This only loads the GA assets if the tracking ID is defined.

**Comment by longhotsummer** [comment]
This doesn't load GA assets unless a GA tracking ID is defined.  Don't know about Piwik.

**Pull request closed without merging**